### PR TITLE
Update to Pond / React Pond 3.0

### PR DESCRIPTION
**Pull request by nicolas-actyx**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -7,16 +7,23 @@ import { Pond } from '@actyx-contrib/react-pond';
...
     </div>
-  )
+  );
 }
 
 ReactDOM.render(
   <React.StrictMode>
-    <Pond loadComponent={<Waiting />}>
+    <Pond
+      loadComponent={<Waiting />}
```
Comment (wwerner) [inline src/index.js:20]: We just made `onError` required in the latest `react-pond`.
Let's add a reloading hander here like in https://github.com/Actyx/advanced-example/blob/master/src/order-management/root.tsx#L7

**Review by wwerner — CHANGES_REQUESTED** [review]
LGTM and works, just add an `onError` to the pond handler, please.

**Review by rkuhn — COMMENTED** [review]
The main purpose of this repo is to show the essence of what is needed to make todo-react persistent and collaborative. For that reason I deliberately refrained from making stylistic changes — there shouldn’t be anything superfluous in the diff. Now this PR updates the code, but it doesn’t create an updated diff (and doesn’t change the link, either). So I wonder what the best approach would be. Probably taking the same basis as #1 and creating the new minimal diff for the new react-pond release, and then updating the README in a separate PR to point to the new diff.

**Comment by wwerner** [comment]
Ah, right.
So, I'd probably ...

1. Wipe `main`
2. Get the current `mdn/todo-react` 
3. Re-do the changes in a separate branch
4. Make a new PR

I could see to it early next week.
Walked through the changes in an office hour session anyway, should be quick then.

WDYT?

**Comment by rkuhn** [comment]
Sorry for missing this: not sure what you mean by “wipe main” — the desired final state is that `master` equals upstream plus a single commit, where that single commit can easily be inspected and discussed by virtue of being done via a PR.

**Comment by wwerner** [comment]
Sorry, my fault. It's `master` in this repo still.
So, more clearly:
* reset `master` to current `todo-react` state
* rebuild the Pond integration on Actyx v2/Pond v3 in new branch
* make PR from this against `master`

**Comment by wwerner** [comment]
Can be closed in favor of #5 and #6 as discussed ☝️

**Pull request closed without merging**